test/integration_test/basic_test.go

```diff
@@ -187,6 +187,9 @@ func BasicUpgrade(tc *types.TestCase) func(*testing.T) {
 				// Set defaults
 				portworx.SetPortworxDefaults(cluster)
 
+				err = ci_utils.ConstructStorageCluster(cluster, hopURL, specImages)
```

Comment:
are we calling this to update the env variables? then we could replace `cluster.Spec.Image = specImages["version"]` with `err = ci_utils.ConstructStorageCluster(cluster, hopURL, specImages)`, it's safer to call `SetPortworxDefaults` afterwards.